data/campaigns/Sceptre_of_Fire/scenarios/4t_The_Jeweler.cfg

```diff
@@ -300,7 +300,7 @@
         [/message]
         [message]
             speaker=Theganli
-            message= _ "Well... Maybe you can ask the Shorbear clan? They have good tools... Chisels and grit of some rare mineral, I don’t know what it is or how they got it, but their work is well known among us gem crafters."
+            message= _ "Well... Maybe you can ask the Shorbear clan? They have good tools... chisels made of some rare mineral. I don’t know what mineral it is or how they got it, but their work is well known among us gem crafters."
```

Comment:
"Well... Maybe you can ask the Shorbear clan? They have good tools... chisels made of a metal that could only be worked with the heat of a volcanic forge. I don’t know how much of that legend is true, but their work is well known among us gem crafters."

Too late for the current PR, but it seems a good place to discuss possible future PRs. I'm thinking we could replace the unobtainable mineral with foreshadowing.